src/sbi_cove.adoc

```diff
@@ -824,11 +830,18 @@ struct tsm_info {
 .COVE TSM Capabilities
 [cols="3,3", width=90%, align="center", options="header"]
 |===
-| Bit | Name                               | Definition
-| 1   | COVE_TSM_CAP_PROMOTE_TO_TVM        | TSM supports single-step TVM creation.
-| 2   | COVE_TSM_CAP_LOCAL_ATTESTATION     | TSM supports local attestation.
-| 3   | COVE_TSM_CAP_AIA                   | TSM supports RISC-V AIA
-| 4   | COVE_TSM_CAP_DYN_PAGE_CONVERSION   | TSM supports dynamic page conversion.
+| Bit   | Name                               | Value              | Definition
+| 0     | COVE_TSM_CAP_PROMOTE_TVM           | `0`                | TSM supports multi-step TVM creation.
+| 0     | COVE_TSM_CAP_PROMOTE_TVM           | `1`                | TSM supports single-step TVM creation.
+| 1     | COVE_TSM_CAP_ATTESTATION_LOCAL     | `0`                | TSM does not support local attestation.
+| 1     | COVE_TSM_CAP_ATTESTATION_LOCAL     | `1`                | TSM supports local attestation.
+| 2     | COVE_TSM_CAP_ATTESTATION_REMOTE    | `0`                | TSM does not support remote attestation.
+| 2     | COVE_TSM_CAP_ATTESTATION_REMOTE    | `1`                | TSM supports remote attestation.
+| 3     | COVE_TSM_CAP_AIA                   | `0`                | TSM supports legacy interrupt handling.
+| 3     | COVE_TSM_CAP_AIA                   | `1`                | TSM supports RISC-V AIA.
```

Comment:
Add two lines here 
| 4 | COVE_TSM_CAP_MRFI.       | '1'  | TSM supports RISC-V AIA with MRFI
| 4 | COVE_TSM_CAP_MRFI.       | '0' | TSM supports RISC-V AIA without MRFI

putting these lines here changes the next bit to 5 and the reserved to 6 ...63

Reply:
I modified it accordingly.